lib/lighthouse/benefits_claims/service.rb

```diff
@@ -145,7 +145,7 @@ def prepare_submission_body(body)
 
       # LH PDF generator service crashes with having an empty array for confinements
       # removes confinements from the request body if confinements attribute empty or nil
-      remove_empty_array(body, 'serviceInformation', 'confinements')
+      # remove_empty_array(body, 'serviceInformation', 'confinements')
```

Comment:
It is working now. I left it just commented as a way to track workarounds we had to use + be able to more easily test for regressions. I see us cleaning this up once LH/TE has gone live.

(I removed the herbicide service dates workaround completely because they fixed it before this PR merged to master)

Reply:
👍